web-module/src/main/webapp/resources/structEditJS/dateTime/celements-date-time-field.js

```diff
@@ -449,11 +433,65 @@
       this.#addInputFields();
       this.#addPickerIcons();
       this.#dateTimeFieldController.initFields();
+      this.#connectInterdependence();
+    }
+
+    #connectInterdependence() {
+      this.#connectedWrapper = this.#interdependenceWrapper;
+      if (this.#connectedWrapper) {
+        // observe wrapper for newly connected components
+        this.#connectedWrapper.observe('celDateTime:connected', this.#handleConnectionBind);
+        // observe being hailed by existing components upon connection
+        this.celObserve('celDateTime:hail', this.#handleHailingBind);
+        this.fire('celDateTime:connected');
+        this.fireUpdated();

```

Comment:
1. `connected` establishes the required observation structure between components. `updated` communicates it's current min/max values to listening components. since a component may connect with inital min/max values set we need to fire `updated` after connecting. 
2. components listen on the wrapper for `connected` events, thus it needs to bubble and we have to use `fire`. components listen directly on other components for a `disconnected`event, thus we can use `celFire`.

Reply:
👍